app/models/zendesk/zendesk_ticket.rb

```diff
@@ -80,7 +80,7 @@ def base_attribute_snippets
 
     def has_value?(param, target = nil)
       target ||= @request
-      target.respond_to?(param) and not target.send(param).blank?
+      target.respond_to?(param) && !target.send(param).blank?
```

Comment:
I think we can squash this commit down with the automatic lint commit - there's no real need to distinguish them, particularly as these manual changes look like the kind of thing that could have been automatic.